### PR TITLE
[Enhancement] modify start and stop scripts for cn gracefully exit

### PR DESCRIPTION
**Pull request by choury**
## What type of PR is this：
- [ ] bug
- [ ] feature
- [x] enhancement
- [ ] refactor
- [ ] others

## Which issues of this PR fixes ：

Fixes #

## Problem Summary(Required) ：

1. Add graceful option for `stop_cn.sh`
2. Make starrocks run as pid 1 in container, let CN handle signal from container manager.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [x] I have added user document for my new feature or new function

### CODE REVIEW

**Review by imay — APPROVED** [review]

**Review by adzfolc — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

File: bin/start_cn.sh [hunk 1]
```diff
@@ -125,5 +125,5 @@ fi
 if [ ${RUN_DAEMON} -eq 1 ]; then
     nohup ${STARROCKS_HOME}/lib/starrocks_be --cn "$@" >> $LOG_DIR/cn.out 2>&1 </dev/null &
 else
-    ${STARROCKS_HOME}/lib/starrocks_be --cn "$@" >> $LOG_DIR/cn.out 2>&1 </dev/null
+    exec ${STARROCKS_HOME}/lib/starrocks_be --cn "$@" >> $LOG_DIR/cn.out 2>&1 </dev/null
```
Comment (kangkaisen) [inline bin/start_cn.sh:128]: why need to add exec here? why start_be.sh needn't it?

Comment (choury) [reply bin/start_cn.sh:128]: Make starrocks_be run as pid 1 in container (instead of child of bash) , then it will receive signal from docker.

**Review by kangkaisen — APPROVED** [review]

**Comment by choury** [comment]
@Mergifyio backport branch-2.4

**Comment by Linkerist** [comment]
@Mergifyio backport branch-2.3

**Pull request merged**